firstvoices/backend/search/query_builder.py

```diff
@@ -33,4 +35,14 @@ def get_search_query(q=None, site_slug=None):
     if site_slug:
         search_query = search_query.query(get_site_filter_query(site_slug))
 
+    # Adding type filters
+    # If only one of the "words" or "phrases" is present, we need to filter out the other one
+    # no action required if both are present
```

Comment:
Minor, but I think all this conditional logic could go in a get_types_query function.